app/models/conference.rb

```diff
@@ -21,6 +21,7 @@ def tickets_left
   private
 
     def date_interval
+      return unless starts_on.present?
```

Comment:
If I delete this line (in previous commit) - because it is not needed for the app to run properly - there are 4 test that fail.
```rspec ./spec/controllers/orga/conferences_controller_spec.rb:18 # Orga::ConferencesController with admin logged in POST create creates a record
rspec ./spec/controllers/orga/conferences_controller_spec.rb:25 # Orga::ConferencesController with admin logged in POST create sets the current season
rspec ./spec/models/conference_spec.rb:9 # Conference should require round to be set
rspec ./spec/models/conference_spec.rb:8 # Conference should require name to be set
```
I can't figure out why?